src/ansys/tools/path/path.py

```diff
@@ -1088,57 +1088,49 @@ def get_mechanical_path(
     return _get_application_path("mechanical", allow_input, version, find)
 
 
-def _mechanical_version_from_path(path: str) -> int:
-    """Extract the Ansys Mechanical version from a path.
-
-    Generally, the version of Mechanical is contained in the path:
-
-    - On Windows, for example: ``C:/Program Files/ANSYS Inc/v231/aisol/bin/winx64/AnsysWBU.exe``
-    - On Linux, for example: ``/usr/ansys_inc/v231/aisol/.workbench``
+def _version_from_path(path: str, product_name: str, path_version_regex: str) -> int:
+    """Extract the version from the executable path.
 
     Parameters
     ----------
-    path : str
-        Path to the Mechanical executable file.
+    path: str
+        The path to the Ansys executable. For example:
 
-    Returns
-    -------
-    int
-        Integer version number (for example, 231).
+        Mechanical:
+        - Windows: ``C:/Program Files/ANSYS Inc/v231/aisol/bin/winx64/AnsysWBU.exe``
+        - Linux: ``/usr/ansys_inc/v231/aisol/.workbench``
```

Comment:
While we are at it, I would update the path to a more recent version. But feel free to discard.
```suggestion
        - Windows: ``C:/Program Files/ANSYS Inc/v251/aisol/bin/winx64/AnsysWBU.exe``
        - Linux: ``/usr/ansys_inc/v251/aisol/.workbench``
```